### PR TITLE
Converting DatasetOwner to hibernate entity

### PR DESCRIPTION
**Pull request by ShridharSattur**
Making FlowKeys Implementing serializable and also converting dataset owner to hibernate entity

### CODE REVIEW

File: wherehows-dao/src/main/java/wherehows/dao/DatasetsDao.java [hunk 1]
```diff
@@ -85,14 +85,12 @@
   private static final String GET_DATASAT_COMPLIANCE_BY_DATASET_ID =
       "SELECT dataset_id, dataset_urn, compliance_purge_type, compliance_entities, confidentiality, "
           + "dataset_classification, field_classification, record_owner_type, retention_policy, "
-          + "geographic_affinity, modified_by, modified_time "
-          + "FROM dataset_compliance WHERE dataset_id = ?";
+          + "geographic_affinity, modified_by, modified_time " + "FROM dataset_compliance WHERE dataset_id = ?";
```
Comment (alyiwang) [inline wherehows-dao/src/main/java/wherehows/dao/DatasetsDao.java]: In this case, don't do reformat, as the "FROM" sentence is more clear at the next line.

File: wherehows-dao/src/main/java/wherehows/dao/DatasetsDao.java [hunk 2]
```diff
@@ -85,14 +85,12 @@
...
 
   private static final String GET_DATASET_COMPLIANCE_BY_URN =
       "SELECT dataset_id, dataset_urn, compliance_purge_type, compliance_entities, confidentiality, "
           + "dataset_classification, field_classification, record_owner_type, retention_policy, "
-          + "geographic_affinity, modified_by, modified_time "
-          + "FROM dataset_compliance WHERE dataset_urn = ?";
+          + "geographic_affinity, modified_by, modified_time " + "FROM dataset_compliance WHERE dataset_urn = ?";
```
Comment (alyiwang) [inline wherehows-dao/src/main/java/wherehows/dao/DatasetsDao.java]: Same as above

File: wherehows-dao/src/main/java/wherehows/dao/DatasetsDao.java (hunk 1 above)
Comment (ShridharSattur) [inline wherehows-dao/src/main/java/wherehows/dao/DatasetsDao.java]: Fixed

File: wherehows-dao/src/main/java/wherehows/dao/DatasetsDao.java (hunk 2 above)
Comment (ShridharSattur) [inline wherehows-dao/src/main/java/wherehows/dao/DatasetsDao.java]: Fixed

**Pull request merged**